src/program/services/scrapers/prowlarr.py

```
@@ -375,18 +377,40 @@ def scrape_indexer(self, indexer: Indexer, item: MediaItem) -> dict[str, str]:
+
+                for future in concurrent.futures.as_completed(future_to_torrent):
+                    torrent, title = future_to_torrent[future]
+                    try:
+                        infohash = future.result(timeout=30)
+                        if infohash:
+                            streams[infohash] = title
+                    except concurrent.futures.TimeoutError:
+                        logger.debug(
+                            f"Timeout getting infohash from downloadUrl for {title}"
+                        )
+                    except Exception as e:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)